.ci/jsoref-spellchecker/whitelist.words

```diff
@@ -80,6 +80,7 @@ ation
 atlassian
 attlist
 attr
+auml
```

Comment:
please remove from whitelist.words all extra words (only `orderedproperties` is ok to stay), please use good words in testing

Reply:
Done.